Add support for Croparia and Stemaria support

shanks3042 opened the pull request:
Both mods didn't inherit from AliasedBlockItem/CropBlock/StemBlock thus they do not work with this mod.
I just added a string comparison for the class name so you don't need any of these mods to compile the project.

Commit:
add support for croparia seeds

Commit:
add stemaria support

emirik0l reviewed (commented):
Commit 3165839 seems unfinished; STEMARIA_IDENTIFIER is identical to CROPARIA_IDENTIFIER and is not checked in isModSeed(). Otherwise, this looks good & I'd be happy to merge with a few minor changes.

emirik0l committed:
Remove STEMARIA_IDENTIFIER

This variable is redundant and doesn't appear to be necessary for stemaria to be supported

emirik0l committed:
Rename ModSeed and isModSeed()

For future maintainability
